src/Test/Utilities/CoreClr/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.Test.Resources.Proprietary": "2.0.0-pre-20160714",
+    "Microsoft.CSharp": "4.3.0",
+    "Microsoft.NETCore.Platforms": "1.1.0",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+    "System.Collections.Concurrent": "4.3.0",
+    "System.Console": "4.3.0",
+    "System.Diagnostics.Debug": "4.3.0",
+    "System.Diagnostics.Tools": "4.3.0",
+    "System.Diagnostics.TraceSource": "4.3.0",
+    "System.IO.FileSystem": "4.3.0",
+    "System.Reflection.TypeExtensions": "4.3.0",
+    "System.Text.RegularExpressions": "4.3.0",
+    "System.Threading.Thread": "4.3.0",
+    "System.Xml.XDocument": "4.3.0",
+    "System.Xml.XmlDocument": "4.3.0",
+    "xunit": "2.2.0-beta4-build3444",
```

Comment:
Why does the utilities library need to reference `xunit` (or at the very least, why does it need to reference the xunit runners)?